tla/tlc.sh

```diff
@@ -4,10 +4,8 @@
 # Original License below
 # Adapted from: https://github.com/pmer/tla-bin
 
-TLC_OPTIONS=()
-
 if [ "${CI}" ] || [ "${SYSTEM_TEAMFOUNDATIONCOLLECTIONURI}" ]; then
-    JVM_OPTIONS=("-Dtlc2.TLC.ide=Github" "-Dutil.ExecutionStatisticsCollector.id=be29f6283abeed2fb1fd0be898bc6601")
+    JVM_OPTIONS+=("-Dtlc2.TLC.ide=Github" "-Dutil.ExecutionStatisticsCollector.id=be29f6283abeed2fb1fd0be898bc6601")
```

Comment:
I should have thought of this when I hacked `trace_validation.yml`.

Reply:
No worries, there's a few other tweaks also to make output easier to read:

```
traces/startup.ndjson
Finished in 07s at (2024-02-12 16:03:41)
traces/append.ndjson
Finished in 07s at (2024-02-12 16:03:41)
traces/startup_2nodes.ndjson
Finished in 09s at (2024-02-12 16:03:43)
traces/replicate.ndjson
Finished in 10s at (2024-02-12 16:03:44)
traces/reconnect.ndjson
Finished in 10s at (2024-02-12 16:03:44)
traces/retire_one.ndjson
Finished in 11s at (2024-02-12 16:03:45)
traces/election.ndjson
Finished in 12s at (2024-02-12 16:03:46)
traces/reelection.ndjson
Finished in 14s at (2024-02-12 16:03:48)
traces/reconfiguration.ndjson
Finished in 16s at (2024-02-12 16:03:50)
traces/soft_rollback.ndjson
Finished in 17s at (2024-02-12 16:03:51)
traces/check_quorum.ndjson
Finished in 20s at (2024-02-12 16:03:54)
traces/multi_election.ndjson
Finished in 20s at (2024-02-12 16:03:54)
traces/suffix_collision.1.ndjson
Finished in 12s at (2024-02-12 16:03:55)
traces/election_while_reconfiguration.ndjson
Finished in 24s at (2024-02-12 16:03:58)
traces/bad_network.ndjson
Finished in 26s at (2024-02-12 16:04:00)
traces/fancy_election.1.ndjson
Finished in 47s at (2024-02-12 16:04:21)
traces/fancy_election.2.ndjson
Finished in 02min 07s at (2024-02-12 16:05:41)
traces/suffix_collision.2.ndjson
Finished in 04min 44s at (2024-02-12 16:08:27)
traces/suffix_collision.3.ndjson
Finished in 08min 34s at (2024-02-12 16:12:19)
```

Also the publish artifacts did not really work, because the files were actually saved under build, and that's now fixed.